<!DOCTYPE html>
<html>
	<head>
		<meta charset="utf-8">
		
		<title>CVE-2021-28363 - Vulnerability Data</title>
		
		<link rel="shortcut icon" type="image/png" href="/favicon.ico" />
		<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css">
		  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
		<script src="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/js/materialize.min.js"></script>
		<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-175514417-4"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-175514417-4');
</script>

	</head>
<body>
	<div class="container">
	

  <nav>
    <div class="nav-wrapper">
      <div class="col s12">
        <a href="https://www.vulnerabilitydata.com" class="breadcrumb">Vulnerability Data</a>
        <a href="/2021" class="breadcrumb">2021</a>
        <a href="#" class="breadcrumb">CVE-2021-28363</a>
      </div>
    </div>
  </nav>

<h2>Vulnerability Data</h2>
<h1>CVE-2021-28363</h1>
<p>The urllib3 library 1.26.x before 1.26.4 for Python omits SSL certificate validation in some cases involving HTTPS to HTTPS proxies. The initial connection to the HTTPS proxy (if an SSLContext isn't given via proxy_config) doesn't verify the hostname of the certificate. This means certificates for different servers that still validate properly with the default urllib3 SSLContext will be silently accepted.</p>

<dl>
	<dt>Published</dt>
	<dd>2021-03-15</dd>

	<dt>Modified</dt>
	<dd>2021-10-20</dd>
</dl>

<h3>References</h3>
<ul>
	<li><a href="https://github.com/urllib3/urllib3/commit/8d65ea1ecf6e2cdc27d42124e587c1b83a3118b0">CONFIRM:https://github.com/urllib3/urllib3/commit/8d65ea1ecf6e2cdc27d42124e587c1b83a3118b0</a></li>
<li><a href="https://github.com/urllib3/urllib3/security/advisories/GHSA-5phf-pp7p-vc2r">CONFIRM:https://github.com/urllib3/urllib3/security/advisories/GHSA-5phf-pp7p-vc2r</a></li>
<li><a href="https://pypi.org/project/urllib3/1.26.4/">CONFIRM:https://pypi.org/project/urllib3/1.26.4/</a></li>
<li><a href="https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/4S65ZQVZ2ODGB52IC7VJDBUK4M5INCXL/">FEDORA:FEDORA-2021-3f378dda90</a></li>
<li><a href="https://security.gentoo.org/glsa/202107-36">GENTOO:GLSA-202107-36</a></li>
<li><a href="https://github.com/urllib3/urllib3/commits/main">MISC:https://github.com/urllib3/urllib3/commits/main</a></li>
<li><a href="https://www.oracle.com/security-alerts/cpuoct2021.html">MISC:https://www.oracle.com/security-alerts/cpuoct2021.html</a></li>
</ul>

<footer>
	<p><a href="https://reconmap.org">Track security vulnerabilities</a> with <strong>Reconmap</strong>.</p>
</footer>


	</div>
</body>
</html>